teku/src/main/java/tech/pegasys/teku/cli/options/InteropOptions.java

```diff
@@ -61,6 +62,14 @@ public class InteropOptions {
       description = "Represents the total number of validators in the network")
   private int interopNumberOfValidators = InteropConfig.DEFAULT_INTEROP_NUMBER_OF_VALIDATORS;
 
+  @Option(
```

Comment:
Implementation of this option was dropped. We could have a separate issue for removal.

Reply:
https://github.com/Consensys/teku/issues/9494